From: Justin M. Forbes <jforbes@redhat.com>
Date: Wed, 20 May 2009 16:54:29 -0500
Subject: [xen] sched: remove printk introduced with hard virt
Message-id: 20090520215429.GA26783@redhat.com
O-Subject: [RHEL5.4 PATCH] Remove unnecessary printk introduced with hard virt patch.
Bugzilla: 501475
RH-Acked-by: Don Dutile <ddutile@redhat.com>
RH-Acked-by: Chris Lalancette <clalance@redhat.com>

BZ#501475
https://bugzilla.redhat.com/show_bug.cgi?id=501475

Description:
The hard virt patch introduced a message printed every time we start a
domain.  This message serves no real purpose and just adds to the clutter.

--
Justin M. Forbes

commit 2d17633924dcc4a300dc11fd53c5bb26cc4f8da0
Author: Justin M. Forbes <jforbes@redhat.com>
Date:   Wed May 20 15:35:20 2009 -0500

    Remove unnecessary printk which puts generally useless information on the
    console every time a domain is started.

    Signed-off-by: Justin M. Forbes <jforbes@redhat.com>

diff --git a/common/sched_credit.c b/common/sched_credit.c
index 1224abd..d0c3359 100644
--- a/common/sched_credit.c
+++ b/common/sched_credit.c
@@ -901,8 +901,6 @@ csched_dom_cntl(
         hard_virt = (op->u.credit.weight >> 15) & 0x1;
         op->u.credit.weight &= 0x7fff;
                                                                                                                              
-        printk("Weight assignment %u - w %u h_virt %u\n", d->domain_id,
-                   op->u.credit.weight, hard_virt);
         if (hard_virt != atomic_read(&d->hard_virt))
         {
            if (!hard_virt)
